DNN Platform/Library/Services/FileSystem/FileInfo.cs

```diff
@@ -69,18 +69,18 @@ public FileInfo()
             VersionGuid = Guid.NewGuid();
         }
 
-        public FileInfo(int portalId, string filename, string extension, int filesize, int width, int height, string contentType, string folder, int folderId, int storageLocation, bool cached)
+        public FileInfo(int portalId, string filename, string extension, long filesize, int width, int height, string contentType, string folder, int folderId, int storageLocation, bool cached)
```

Comment:
This creates backward compatibility issue. Must deprecate and create another one.

Reply:
What do you mean by backward compatibility issue?
Long data type is supported by SQL Server since very early versions

Reply:
I mean the existing C# methods/constructors signatures must not change. You must keep the old signatures and create new new ones as needed; something like this:
```
[Obsolete("Deprecated in Platform 9.2.0. Please use the construction with 'long filesize' instead")]
public FileInfo(int portalId, string filename, string extension, int filesize, ...)
    : this(portalId, filename, extension, (long)filesize, ..)
{
  // move the code - if any - to the other constructor
}

public FileInfo(int portalId, string filename, string extension, long filesize, ...) {
// move the other constructor code here
}
```